assemble/src/main/assemblies/component.xml

```diff
@@ -40,15 +40,48 @@
         <include>org.apache.commons:commons-math3</include>
         <include>org.apache.commons:commons-vfs2</include>
         <include>org.apache.thrift:libthrift</include>
+        <include>org.eclipse.jetty:jetty-continuation</include>
         <include>org.eclipse.jetty:jetty-http</include>
         <include>org.eclipse.jetty:jetty-io</include>
         <include>org.eclipse.jetty:jetty-security</include>
         <include>org.eclipse.jetty:jetty-server</include>
+        <!-- jetty-servlet only needed by old monitor -->
         <include>org.eclipse.jetty:jetty-servlet</include>
         <include>org.eclipse.jetty:jetty-util</include>
         <include>org.apache.htrace:htrace-core</include>
         <include>org.slf4j:slf4j-api</include>
         <include>org.slf4j:slf4j-log4j12</include>
+        <!-- Jersey/Jackson-based webservice -->
```

Comment:
I looked at Spark vs Jersey packaging in Linux distributions and I agree with @ctubbsii's point that Jersey is a better choice for packaging.  Jersey is in Fedora and Ubuntu while Spark hasn't been packaged there yet.